decidim-admin/app/views/decidim/admin/taxonomies/index.html.erb

```diff
@@ -0,0 +1,125 @@
+<% add_decidim_page_title(t("taxonomies", scope: "decidim.admin.titles")) %>
+<div class="card" id="areas" role="region" aria-labelledby="taxonomies-title">
+  <div class="item_show__header">
+    <h1 class="item_show__header-title" id="taxonomies-title">
+      <%= t "decidim.admin.titles.taxonomies" %>
+      <% if allowed_to? :create, :taxonomy %>
+        <%= link_to t("new.create", scope: "decidim.admin.taxonomies"), new_taxonomy_path, class: "button button__sm button__secondary new" %>
+      <% end %>
+    </h1>
+  </div>
+  <% if taxonomies.any? %>
+    <div class="filters__section">
+      <div class="fcell search">
+        <%= search_form_for(@query, url: taxonomies_path) do |form| %>
+          <div class="input-group">
+            <%= form.search_field(
+                  :name_cont,
+                  class: "input-group-field bg-secondary/5",
+                  label: false,
+                  placeholder: t("filters.search_placeholder", scope: "decidim.admin.taxonomies")
+                ) %>
+            <div class="input-group-button">
+              <button type="submit" class="text-secondary" aria-label="<%= t("decidim.search.term_input_placeholder") %>">
+                <%= icon "search-line", class: "fill-secondary w-4 h-4" %>
+              </button>
+            </div>
+          </div>
+        <% end %>
+      </div>
+    </div>
+    <div class="table-scroll" role="table" aria-labelledby="taxonomies-title">
+      <table class="table-list">
+        <thead>
+        <tr role="row">
+          <th scope="col" role="columnheader"><span class="sr-only">Move</span></th>
+          <th scope="col" role="columnheader" class="!text-left"><%= t("decidim.admin.taxonomies.name") %></th>
+          <th scope="col" role="columnheader"><%= t("decidim.admin.taxonomies.amount") %></th>
+          <th scope="col" role="columnheader"><%= t("decidim.admin.taxonomies.actions") %></th>
+        </tr>
+        </thead>
+        <tbody class="draggable-list js-connect js-list-available" data-sort-url="<%= taxonomies_path %>" role="rowgroup">
```

Comment:
Is this correct? when sorting it should reorder all the weights for the same parent.